lib/disasters.js

```diff
@@ -145,7 +163,82 @@ function getDisaster(disasterId, callback) {
   });
 }
 
+async.auto({
+  disasters: ['operations', cacheDisasters],
+},
+function (err, results) {
+  if (err) {
+    console.log("hid_profiles cron run failed.");
+  }
+  else {
+    console.log("Finished hid_profiles cron run successfully.");
+  }
+  process.exit(err ? 1 : 0);
+});
+
 
+function newDisasterNotification(disaster) {
+  var requiredLocations = [];  
+  async.series([
+    function (cb) {
+      if(disaster.countries.length > 0){
+        async.forEachSeries(disaster.countries, function(country, callback2) {
+          if (!requiredLocations[iso3Countries[country]]) {
+                requiredLocations[iso3Countries[country]] = {};
+                requiredLocations[iso3Countries[country]].glide = [];
+                requiredLocations[iso3Countries[country]].contacts = [];
+                requiredLocations[iso3Countries[country]].disasterName = [];
+              }  
+            requiredLocations[iso3Countries[country]].glide.push(disaster.glide_id);
+            Contact.find({ $query: { type: 'local', location: iso3Countries[country]},  $orderby : {location: -1} }, function (err, contact){
+              contact.forEach(function(contact){
+                if(!contact){
+                  console.log("No users found");
+                  return cb();
+                }   
+                requiredLocations[iso3Countries[country]].contacts.push(contact);       
+            });   
+            callback2();         
+          })
+        }, cb);
+      }
+    },
+    function (cb){
+      var mailOptions = {};
+      mailOptions.requiredLocations = requiredLocations;
+      mailOptions.baseUrl = process.env.APP_BASE_URL;
+      mailOptions.subject = 'New Disaster in your region';
+      mailOptions.contacts = false;
+
+      if(disaster.countries.length > 0){
+        async.forEachSeries(disaster.countries, function(country, cb3) {
+          if(requiredLocations[iso3Countries[country]].contacts.length > 0)
+            mailOptions.contacts = true;
```

Comment:
There were some cases on my local machine where there were no contacts associated with the disaster country, so it was throwing me an error. I doubt that will be a possibility on prod, so I will remove this